spanner/cloud-client/src/test/java/com/example/spanner/SpannerStandaloneExamplesIT.java

```diff
@@ -100,6 +119,57 @@ public void executeSqlWithCustomTimeoutAndRetrySettings_shouldWriteData() {
     assertThat(out).contains("1 record inserted.");
   }
 
+  @Test
+  public void addNumericColumn_shouldSuccessfullyAddColumn() {
+    String out =
+        runExample(
+            () ->
+                AddNumericColumnSample.addNumericColumn(
+                    spanner.getDatabaseAdminClient(), instanceId, databaseId));
+    assertThat(out).contains("Successfully added column `Revenue`");
+  }
+
+  @Test
+  public void insertNumericData_shouldWriteData() {
+    String projectId = spanner.getOptions().getProjectId();
+    String out =
+        runExample(
+            () ->
+                InsertNumericDataSample.insertNumericData(
+                    spanner.getDatabaseClient(DatabaseId.of(projectId, instanceId, databaseId))));
+    assertThat(out).contains("Records successfully inserted");
+  }
+
+  @Test
+  public void queryWithNumericParameter_shouldReturnResults() {
+    String projectId = spanner.getOptions().getProjectId();
+    DatabaseClient client =
+        spanner.getDatabaseClient(DatabaseId.of(projectId, instanceId, databaseId));
+    client.write(
+        ImmutableList.of(
```

Comment:
Would the table not already having this entries from the previous test?

Reply:
No, JUnit (the Java test framework) does not guarantee the order in which tests are executed, and it is strongly recommended for this and other reasons to create test cases that are completely independent of all other test cases. So this test case cannot rely on any data already being present in the table, as there is no guarantee that this test case is executed after any of the other test cases.

(An exception to the above rule is normally made for heavy weight operations like creating the data model. That is created only once for the entire test file.)